test/unit/math/src/mathComplexDouble.cpp

```diff
@@ -37,9 +37,18 @@ TEMPLATE_LIST_TEST_CASE("mathOpsComplexDouble", "[math] [operator]", TestAccFunc
     testTemplate.template operator()<alpaka::Complex<double>>();
 }
 
-#ifdef __cpp_lib_is_layout_compatible
-TEMPLATE_LIST_TEST_CASE("mathOpsComplexDouble", "[layout]", TestAccs)
+TEMPLATE_LIST_TEST_CASE("mathOpsComplexDouble", "[array-oriented]", TestAccs)
 {
-    STATIC_REQUIRE(std::is_layout_compatible_v<alpaka::Complex<double> std::complex<double>>);
```

Comment:
I think the check for `is_layout_compatible` should be correct. However, it fails with gcc and MSVC: https://godbolt.org/z/vdKeGenGr

Maybe a compiler bug in both cases? Can you please comment out the line instead of removing and add a comment that we should revisit this in the future?

Reply:
See #1720 for the discussion about this.

Reply:
Oh, thx! I guess `layout_compatible` might be too strict indeed.